frontend/src/scenes/session-recordings/sessionRecordingLogic.ts

```diff
@@ -183,7 +198,34 @@ export const sessionRecordingLogic = kea<sessionRecordingLogicType>({
         },
     }),
     selectors: {
-        sessionEvents: [(selectors) => [selectors.sessionEventsData], (eventsData) => eventsData?.events ?? []],
+        sessionEvents: [
+            (selectors) => [selectors.sessionEventsData, selectors.sessionPlayerData],
+            (eventsData, playerData) => {
+                return (eventsData?.events ?? []).map((e: EventType) => ({
+                    ...e,
+                    timestamp: +dayjs(e.timestamp),
+                    zeroOffsetTime:
+                        clamp(
+                            +dayjs(e.timestamp),
+                            playerData.session_recording.start_time,
+                            playerData.session_recording.end_time
+                        ) - playerData.session_recording.start_time,
+                }))
+            },
+        ],
+        eventsToShow: [
+            (selectors) => [selectors.filters, selectors.sessionEvents],
+            (filters, events) => {
+                return filters?.query
+                    ? new Fuse(events, {
+                          threshold: 0.3,
+                          keys: ['event'],
```

Comment:
I think we should allow searching on the displayed event text too, so you can search for "Continue" and see the autocapture event that says 'Clicked span with text "Continue"'

Reply:
good catch! had this in the back of my mind but forgot to implement it